### PR TITLE
Add support of fleet-level task

### PR DESCRIPTION
**Pull request by cwrx777**
Fleet-level task is implemented by ignoring `bid_notice` msg in `bid_notice_cb` if the `fleet_name` in the task_request does not match the fleet's name.

### CODE REVIEW

File: rmf_fleet_adapter/src/rmf_fleet_adapter/agv/FleetUpdateHandle.cpp
Comment (Yadunund) [inline rmf_fleet_adapter/src/rmf_fleet_adapter/agv/FleetUpdateHandle.cpp:518]: Delete unnecessary new lines.

File: rmf_fleet_adapter/src/rmf_fleet_adapter/agv/FleetUpdateHandle.cpp
Comment (Yadunund) [inline rmf_fleet_adapter/src/rmf_fleet_adapter/agv/FleetUpdateHandle.cpp:506]: I think this block can be simplified a bit. Also add some comments and logs.
```suggestion
  //  If a fleet_name was specified in the request, only proceed if the value matches
  // the name of this fleet. 
  if (request_msg.contains("fleet_name") &&
    request_msg["fleet_name"].template get<std::string>() != name)
  {
    RCLCPP_INFO(
      node->get_logger(),
      "Ignoring BidNotice request as it is for fleet %s.,
      request_msg["fleet_name"].template get<std::string>().c_str());
    return;
  } 
```

**Review by Yadunund — CHANGES_REQUESTED** [review]

File: rmf_fleet_adapter/src/rmf_fleet_adapter/agv/FleetUpdateHandle.cpp
Comment (Yadunund) [reply rmf_fleet_adapter/src/rmf_fleet_adapter/agv/FleetUpdateHandle.cpp:519]: @cwrx777 this is still unaddressed.

Comment (cwrx777) [reply rmf_fleet_adapter/src/rmf_fleet_adapter/agv/FleetUpdateHandle.cpp]: @Yadunund 
Done. please revieew

**Review by Yadunund — APPROVED** [review]

**Review by Yadunund — CHANGES_REQUESTED** [review]
Looks like there are some whitespaces on line 506 which is making CI unhappy. Can you format the code? `ament_uncrustify -c ~/ws_rmf/src/rmf/rmf_utils/rmf_utils/test/format/rmf_code_style.cfg ~/ws_rmf/src/rmf/rmf_ros2/rmf_fleet_adapter/src/rmf_fleet_adapter/ --reformat
` should automatically format the code.

**Review by Yadunund — DISMISSED** [review]

**Comment by Yadunund** [comment]
@cwrx777 can you ensure all your commits are signed?

**Comment by Yadunund** [comment]
@cwrx777 this merging is blocked by not having signed commits.

**Comment by Yadunund** [comment]
@cwrx777 looks like something went wrong with the rebase and there are several new commits and diffs. Could you grant me push access to your fork/branch? I can clean this up and get it merged.

**Comment by cwrx777** [comment]
> @cwrx777 looks like something went wrong with the rebase and there are several new commits and diffs. Could you grant me push access to your fork/branch? I can clean this up and get it merged.

@Yadunund, I've granted you contributor access. Please try. Sorry for the trouble.

**Review by Yadunund — APPROVED** [review]

**Pull request merged**
